### ISSUE TITLE
Update muon dashboard to include RB number and PI

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#5468]
As a user on EMU I would like to avoid assigning my data to the wrong RB number and thus making it unavailable to me. Having the RB number and the PI on the IBEX dashboard would make it more obvious when I input this information wrong. (Note the PI is currently there inside the list of users but I would like it separately to highlight it)

## Acceptance Criteria
1. The current RB number is on the IBEX dashboard
1. The PI is separately listed on the dashboard

**Comment by kjwoodsISIS** [comment]
Would this not be more useful for all scientists? (not just muon scientists).

**Comment by DominicOram** [comment]
This came in a direct request from Muons, dashboard space is usually at a premium so others may or may not want it. The ticket should be done with the view that it should be included on some instruments and not others, we can then highlight it to other scientists during demos to ask if they would also like it

**Comment by John-Holt-Tessella** [comment]
Is this the ibex dashboard in the GUI not the web dashboard. Could they add a block?

**Comment by DominicOram** [comment]
Yes, it's the GUI, not the web dashboard. The request was on the dashboard, I'm not sure if a block would suffice, I'll ask. We currently don't have a PV that gives the PI so we would need to do that either way.

**Comment by DominicOram** [comment]
This should be hidden on the web dashboard
